gargi.md

```diff
@@ -0,0 +1,3 @@
+Email:500062353@stu.upes.ac.in
```

Comment:
@Gargi-Jaiswal , the content of the file looks like :- 

``` Contetntttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt```` it's in the markdown code formatting.